test/features/binary_build.feature

```diff
@@ -16,7 +16,7 @@ Feature: Use binary build to deploy the service
     And Start build with name "process-springboot-example-binary" from local example service path "process-springboot-example/target"
 
     Then Kogito application "process-springboot-example" has 1 pods running within 5 minutes
-    And HTTP GET request on service "process-springboot-example" with path "orders" is successful within 2 minutes
+    And Service "process-springboot-example" contains 0 instance of process with name "orders" within 2 minutes
```

Comment:
Could we create a new step for this,  like `Service "process-springboot-example" with process name "orders" is available within 2 minutes` ?
that would the confusion with `0 instance of process`. we just want to know if service is avaible here.

Reply:
Done